injected/src/services/fs.ts

```diff
@@ -42,6 +42,12 @@ export class FS extends Service {
     return (await getRes()).data.trim();
   }
 
+  removeFile(path: string) {
+    info('removeFile', path);
+
+    return rpcRequest<{ path: string }, {}>('FSService.RemoveFile', { path });
```

Comment:
Right now it doesn't work because `getRes()` is never called. You could change `install-plugin.ts` to run `smm.FS.removeFile(...).getRes()`, but in this case I would change this method to just return `getRes()` (that's how it's usually done, the `untar` method is an exception).

```suggestion
    const { getRes } = rpcRequest<{ path: string }, {}>(
      'FSService.RemoveFile',
      { path }
    );
    return getRes();
```

Reply:
Ahhhh, okay. Yeah, I saw what was going on there and copied it. Good to know. Will make the changes here shortly :)